Services/DataX.Utilities/DataX.Utilities.Web/RolesCheck.cs

```diff
@@ -24,6 +29,25 @@ public static void EnsureWriter(HttpRequest request, bool isLocal)
                 EnsureWriter(request);
             }
         }
+        private static void ScenarioTestHelper()

```

Comment:
Not sure about the name of this or the file it is in.  The header is going to help i think, can this be more generic or in its own test setupclass?

Reply:
I will rename this to AddWhitelistedTestClientUserId()

Reply:
Updated